lua/orgmode/objects/calendar.lua

```diff
@@ -150,10 +150,9 @@ function Calendar:render()
   vim.api.nvim_set_option_value('modifiable', true, { buf = self.buf })
 
   local cal_rows = { {}, {}, {}, {}, {}, {} } -- the calendar rows
-  local start_from_sunday = config.calendar_week_start_day == 0
```

Comment:
@kristijanhusak It seems, that this changes to the config introduced a regression to how the date mapping tests work. But I currently don't see why ...